main.py

```diff
@@ -146,6 +147,8 @@ def run_checks():
     inf(f"* Voting month:  {voting_month_year_str}")
     inf(f"* Current month: {current_month_year_str}")
 
+    prompt_for_missing_durs(ensure_duration_var.get())
```

Comment:
As mentioned earlier, this `prompt_for_missing_durs` function would need to be called in any other applications which are using the fetch service. As a reminder, there are currently 3 applications:

* `main.py` (this one, which analyzes all the votes and extracts video information)
* `calc_main.py` (takes in the votes after manual inspection by Littleshy and calculates the top 10 for the month)
* `post-processing.py` (formats the top 10 videos in ways that are useful for the final showcase)

`post-processing.py` also use the fetch services, so it would also need a `prompt_for_missing_durs` call.

An alternative way to do this would be to update the `get_fetcher` method, which returns the fetcher used by the application. This is where the various fetch services are set up, so this is a perfect time to pass them information such as `prompt_dur`. If you provide appropriate default settings, then other applications won't need to be updated and will work as normal.

Reply:
I didn't consider the other applications before oops. I made new commits to fix those issues like you suggested :)